.github/workflows/clear-deploy.yml

```diff
@@ -0,0 +1,24 @@
+name: Clear deployments for closed pull requests
```

Comment:
I think we should standardize whether or not we mean "clear deployments" or "delete deployments". I am okay with using either, but would be best if file names and terminology used are all consistent with one another.

Reply:
I've changed everything to 'delete' and renamed `delete-deployments.sh` to `delete-deploy.sh` for brevity and for similarity with `delete-deploy.yml`